jenkins-slaves/ansible/data/config/pulp/pulp.repo

```diff
@@ -0,0 +1,43 @@
+[rhel-7-server-os-rpms]
+metadata_expire = 86400
+baseurl = https://pulp04.web.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/$releasever/$basearch/os
```

Comment:
It seems that we reveal internal URL here. Probably we should avoid this. Maybe we can inject this as a variable?

Reply:
I personally don't think this is an issue. But of course it can be replaced by variable.

Reply:
For me personally it's not an issue too however I know that CE team is very serious about it. I don't know if this is policy or just CE team practice.

I think we can live with that. And if there is a sense in changing this maybe it's a good task for us to get some practice with the procedure?